Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	pre-commit clean || true
 
 install-pip-setuptools:
-	python -m pip install -U "pip~=21.2" "setuptools>=38.0" wheel
+	python -m pip install -U "pip>=21.2" "setuptools>=38.0" wheel
```

Comment:
I'm still not clear how this is fine on python 3.6 if we install piptools 6.4.

Reply:
This is a great question... The reason (which I figured out last week but then completely forgot about) is that [pip 22 drops support for Python 3.6](https://pip.pypa.io/en/stable/news/#v22-0). So on the 3.6 environment we'll get piptools 6.4 and pip 21.x, which is all good.

Reply:
Aaah that makes sense! Wow nice one, great digging. 👏